### PR TITLE
Proof reading and minor issues correction

### PR DESCRIPTION
**Pull request by ACyprus**
In this pull request, I have:

- corrected minor issues, typos, duplications and broken links
- not addressed general grammar, use of commas etc
- included each change via a separate commit to make cherry picking easier
- noted any assumptions made when unsure in the commit description

### CODE REVIEW

File: general/index.html [hunk 1]
```diff
@@ -158,14 +158,14 @@
               
                 <h1 id="general-commands">General Commands<a class="headerlink" href="#general-commands" title="Permanent link">&para;</a></h1>
 <h2 id="configuration">Configuration<a class="headerlink" href="#configuration" title="Permanent link">&para;</a></h2>
-<p>The base profile provides voice commands for changing the profiles settings.
+<p>The base profile provides voice commands for changing the profile’s settings.
```
Comment (alterNERDtive) [inline general/index.html:161]: This was actually correct/intended; if you want to change it, then to `profiles’`.

File: requirements/index.html [hunk 2]
```diff
@@ -179,7 +179,7 @@ <h2 id="eddi">EDDI<a class="headerlink" href="#eddi" title="Permanent link">&par
 Dangerous, providing responses to events that occur in-game using data from the
 game as well as various third-party tools. In this case, you will need to run it
 as a VoiceAttack plugin.</p>
-<p>EDDI also regularly publishes beta versions. Unless a profiles release
+<p>EDDI also regularly publishes beta versions. Unless a profile’s release
```
Comment (alterNERDtive) [inline requirements/index.html:182]: Same here, that’s intended :)

Comment (ACyprus) [reply requirements/index.html:182]: this one had no apostrophe though, and it implies ownership.

File: general/index.html (hunk 1 above)
Comment (ACyprus) [reply general/index.html:161]: Cool, the main change was due to the dodgy  character

File: requirements/index.html (hunk 2 above)
Comment (alterNERDtive) [reply requirements/index.html:182]: No, it’s a profiles release. Like, uh, a bombs release or something. It’s just plural.

File: general/index.html (hunk 1 above)
Comment (alterNERDtive) [reply general/index.html:161]: That’s not dodgy, that’s a proper apostrophe! You heathen.

Comment (alterNERDtive) [reply general/index.html:161]: > this one...

Funny, doesn’t even show up on Github. Yeah, IDK what that is.

**Pull request closed without merging**